lib/screens/edit_note_page.dart

```diff
@@ -118,24 +117,22 @@ class _EditNotePageState extends State<EditNotePage> {
           ),
           SizedBox(
             height: sizedBoxHeightPallete,
-            child: Expanded(
-              child: ListView.builder(
-                scrollDirection: Axis.horizontal,
-                itemCount: colorPallete.length,
-                itemBuilder: (context, index) {
-                  return GestureDetector(
-                    onTap: () {
-                      _colorTap = index;
-                      // ignore: no-empty-block
-                      setState(() {});
-                    },
-                    child: CircleTap(
-                      color: index,
-                      circleTap: _colorTap,
-                    ),
-                  );
-                },
-              ),
+            child: ListView.builder(
+              scrollDirection: Axis.horizontal,
+              itemCount: colorPallete.length,
+              itemBuilder: (context, index) {
+                return GestureDetector(
+                  onTap: () {
+                    _colorTap = index;
+                    // ignore: no-empty-block
+                    setState(() {});
```

Comment:
State mutations should happen _inside_ the setState callback: https://api.flutter.dev/flutter/widgets/State/setState.html
```suggestion
                    setState(() {
                              _colorTap = index;
                    });
```